OrbitVulkanLayer/include/OrbitVulkanLayer/DispatchTable.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_LAYER_DISPATCH_TABLE_H_
```

Comment:
The directory is `OrbitVulkanLayer`, but the include guards are `ORBIT_LAYER`. I think that Vulkan should also be mentioned in the include guards, e.g, `ORBIT_VULKAN_LAYER` or `ORBIT_VK_LAYER`.

Reply:
+1 (also commented the same elsewhere)

Reply:
renamed to ORBIT_VULKAN_LAYER_...